src/index.html

```diff
@@ -1,12 +1,47 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8" >
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" >
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      rel="stylesheet"
+      href="./styles/main.scss"
+    >
   </head>
   <body>
-    <h1>Calendar</h1>
+    <ul class="calendar">
```

Comment:
Please add this two modifiers for calendar and depending on them display the necessary days
![image](https://user-images.githubusercontent.com/54936044/229475700-b4be01b5-4655-4825-b5e0-39d329efba3d.png)

Reply:
done